bblfsh/client.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+
+import grpc
+
+# The following two insertions fix the broken pb import paths
+sys.path.insert(0, os.path.join(os.path.dirname(__file__),
+                                "github/com/bblfsh/sdk/protocol"))
+sys.path.insert(0, os.path.dirname(__file__))
+from bblfsh.github.com.bblfsh.sdk.protocol.generated_pb2 import ParseUASTRequest
+from bblfsh.github.com.bblfsh.sdk.protocol.generated_pb2_grpc import ProtocolServiceStub
+
+
+class BblfshClient(object):
```

Comment:
I think this should have a method to close the client and the underlying connection.
Maybe `close()`? I'm not sure that's Python's convention. Also, `__enter__` and `__exit__` so it works with `with`?